docs/changes.rst

```diff
@@ -16,6 +16,10 @@ Released: not yet
 
 * Removed support for Python 2.7, 3.6, and 3.7. (issue #1390)
 
+* Removed setup.py and therefore the possibility to run setup.py as a command,
+  for example to install or test pywbemtools. Running setup.py as a command has
+  been deprecated by Python.
+
```

Comment:
Note also that this change added the toml file. i.e. removed setup.py in favor of project.toml

Reply:
The addition of pyproject.toml is stated in a separate change log entry in section "Enhancements". This entry here is about the incompatibility caused by removing setup.py. Nevertheless, changed the entry to:
```
* The migration from setup.py to pyproject.toml removed therefore the
  possibility to run setup.py as a command, for example to install or test
  pywbemtools. Note that running setup.py as a command has been deprecated by
  the Python setuptools team.
```